SecretSanta/test/SecretSanta.Web.UITests/UsersPageTests.cs

```diff
@@ -0,0 +1,143 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using SecretSanta.Web.UITests.TestPages;
+using System;
+using System.Collections.Generic;
+using System.IO;
+//using Microsoft.Extensions.Configuration;
+
+namespace SecretSanta.Web.UITests
+{
+    [TestClass]
+    public class UsersPageTests
+    {
+        private const string RootUrl = "https://localhost:44334/";
+        private IWebDriver Driver { get; set; }
+        public TestContext TestContext { get; set; }
+        [TestInitialize]
+        public void Init()
+        {
+            Driver = new ChromeDriver(Path.GetFullPath("."));
+        }
+
+        [TestCleanup]
+        public void Cleanup()
+        {
+            if (TestContext.CurrentTestOutcome == UnitTestOutcome.Failed)
+            {
+                string projectRoot =
+                    Environment.CurrentDirectory.Substring(0, Environment.CurrentDirectory.LastIndexOf("bin"));
+
+                string fileName = $"{projectRoot}/Screenshots/{TestContext.TestName}.png";
+
+                Screenshot screenshot = ((ITakesScreenshot) Driver).GetScreenshot();
```

Comment:
Though this works with the current driver it would be better to type check the Driver rather than directly casting. Then if you ever use a driver that doesn't support screenshots it wont cause your test to fail.
```suggestion
                if (Driver is ITakesScreenshot takeScreenshot)
                {
                    Screenshot screenshot = takeScreenshot.GetScreenshot();
```